Source/Scene/SkyAtmosphere.js

```diff
@@ -79,38 +81,20 @@ define([
         this._spSkyFromSpace = undefined;
         this._spSkyFromAtmosphere = undefined;
 
-        this._fCameraHeight = undefined;
-        this._fCameraHeight2 = undefined;
-        this._outerRadius = Cartesian3.maximumComponent(Cartesian3.multiplyByScalar(ellipsoid.radii, 1.025, new Cartesian3()));
-        var innerRadius = ellipsoid.maximumRadius;
-        var rayleighScaleDepth = 0.25;
+        // camera height, outer radius, inner radius, dayNight flag
+        this._camAndRadiiAndDayNight = new Cartesian4();
```

Comment:
Usually we try not to abbreviate names, so name it `camera` instead. Also as suggested by @pjcozzi there may be a better name for DayNight. Possibly `dynamicLighting`?